pkg/controllers/node/termination/terminator/events/events.go

```diff
@@ -56,6 +56,16 @@ func NodeFailedToDrain(node *corev1.Node, err error) events.Event {
 	}
 }
 
+func NodeAwaitingVolumeDetachmentEvent(node *corev1.Node) events.Event {
+	return events.Event{
+		InvolvedObject: node,
+		Type:           corev1.EventTypeNormal,
+		Reason:         "AwaitingVolumeDetachment",
+		Message:        "Awaiting deletion VolumeAttachments bound to node",
```

Comment:
Is it useful to list out the volume detachments that we are waiting on (or maybe a pretty list of them) in this list that we have here

Reply:
I didn't for a similar reason that I didn't include the pods on the `Drained` status condition - we would either be hammering the API server as the list changes or the information would be out of date frequently. The former

Reply:
> hammering the API server as the list changes or the information would be out of date frequently

You can make it so that the events are only fired at a certain frequency and they are deduped without considering their message -- honestly, I think including the extra info could be helpful and, if there are ones that are actually stuck, would be really valuable information for a user to know